core/src/main/java/com/novoda/landingstrip/Notifier.java

```diff
@@ -25,6 +28,13 @@ private void recreateAndBindTabs(BaseAdapter<T> adapter) {
         }
     }
 
+    private void handleAdapterSetBecausePageSelectedIsNotCalled(BaseAdapter<T> adapter) {
+        if (state.firstTimeAccessed() && adapter.getCount() > 0) {
```

Comment:
the `adapter.getCount() > 0` check could fit in `onNotifyDataSetChanged()`, then this method would be simpler.